src/fileformats/ocd_file_export.cpp

```diff
@@ -1975,6 +1979,46 @@ void OcdFileExport::setupTextSymbolFraming(const TextSymbol* text_symbol, OcdTex
 }
 
 
+/**
+ * Returns true if the symbol may represent a line of secondary points.
+ */
+bool OcdFileExport::maybeSecondaryPoints(const Symbol* const symbol) const
+{
+	if (symbol->getType() != Symbol::Line)
+		return false;
+	
+	auto* line_symbol = static_cast<const LineSymbol*>(symbol);
+	if (line_symbol->getColor()
+	    || line_symbol->getLineWidth()
+	    || line_symbol->hasBorder())
+		return false;
+	
+	auto const is_empty = [line_symbol](auto getter) -> bool {
+		auto* symbol = (line_symbol->*getter)();
+		return !symbol || symbol->isEmpty();
+	};
+	if (is_empty(&LineSymbol::getMidSymbol))
+		return false;
+	if (!is_empty(&LineSymbol::getStartSymbol))
+		return false;
+	if (!is_empty(&LineSymbol::getDashSymbol))
+		return false;
+	if (!is_empty(&LineSymbol::getEndSymbol))
+		return false;
+	
+	return true;
```

Comment:
I'd be fine with a more compact:
```
if (is_populated(&LineSymbol::getEndSymbol)
    || is_populated(&LineSymbol::getStartSymbol)
    || is_populated(&LineSymbol::getDashSymbol))
        return false;

return is_populated(&LineSymbol::getMidSymbol);
```

Which better expresses the qualification criteria for a possible secondary symbol representation.